src/bin/rdcore/cmdline.rs

```diff
@@ -143,12 +153,16 @@ pub fn parse_args() -> Result<Config> {
 
 fn parse_kargs(matches: &ArgMatches) -> Result<Config> {
     // we could enforce these via clap's ArgGroup, but I don't like how the --help text looks
-    if !(matches.is_present("boot-device") || matches.is_present("boot-mount")) {
+    if !(matches.is_present("boot-device")
+        || matches.is_present("boot-mount")
+        || matches.is_present("override-options"))
+    {
```

Comment:
Maybe also say `// --override-options is undocumented` to prevent someone from updating the `bail!()`?